modules/bitgo/scripts/publish.sh

```diff
@@ -45,8 +45,9 @@ git checkout -q "$REF_NAME"
 PACKAGE_NAME="$(package_json 'name')"
 PACKAGE_VERSION="$(package_json 'version')"
 
-# install
-npm install
+# install - generate package-lock.json first, then use npm ci for reproducible builds
+npm install --package-lock-only
```

Comment:
Doesn't regenerating a package-lock.json file first defeat the purpose of running `npm ci`? Why would the package-lock.json file be incomplete at the time of publishing?